server/src/routers/configRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = new express.Router();
+
+const { mapping } = require('../constants/config');
+
+router.use(express.json());
+
+//Endpoint for mapping
+router.get('/api/specification/mapping', async (req, res) => {
```

Comment:
We aren't using `await` here, right? Can we remove `async` here then?

Reply:
@pcgamer1 maybe this comment got automatically resolved when you pushed. That is why push small changes as one commit, please. Everything under the sun is resolved now